packages/Breadcrumbs/src/components/Link/Link.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Popover from "@paprika/popover";
+import IsDarkContext from "../../context";
+import { MAXIMUM_NUM_OF_CHARACTER } from "../../constants";
```

Comment:
🤔  I don't think this should be part of the constants package, is too specific to the Breadcrumb just add an extra dependency that is not required IMO.